src/Main.java

```diff
@@ -1,8 +1,38 @@
-class MyClass {
-    int year = 2023;
-    public static int myMethod(int a, int b){
-        int c = a + b;
-        return c;}
-    public static void main(String[] args) {
-        myMethod(num1, num2);}
-}
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String args[]) {
```

Comment:
According to the task it was needed to create 3 overloaded methods. Please divide your code
Scanner was needed for 2.3 only